cerebrium/development/serve.mdx

```diff
@@ -11,50 +11,67 @@ This allows you to run your deployment on a dedicated server, and see the result
   This feature is currently in beta and is available to all users. As such, you
   may encounter bugs or limitations. We are actively working on improving the
   experience and adding more features. If you have any feedback or suggestions,
-  we'd love to hear from you on discord or slack!
+  we'd love to hear from you on [Discord](https://discord.gg/ATj6USmeE2) or [Slack](https://join.slack.com/t/cerebriumworkspace/shared_invite/zt-1qojg3eac-q4xyu5O~MeniNIg2jNeadg)!
 </Note>
 
+**Limitations:**
+
+- Build process (packages, apt, etc.) changes require a full restart of the served instance
+- Serve sessions are not cached meaning a full build process is done even when environments have not changed between sessions
+- Reponses are not returned via the Local API server
+
 ## Usage
 
-To start a served instance, first navigate to the root folder of your **cortex deployment**.
+To start a served instance, first navigate to the root folder of your cortex deployment.
 
 Then, simply run the following command in your terminal:
 
 ```bash
 cerebrium serve start
 ```
 
-After running the `cerebrium serve start` command, you will see output in your terminal indicating that the server is starting up. This output will include a URL that you can use to access your deployment locally.
-
-Here are some additional commands that can be useful when working with `cerebrium serve`:
+After running the `cerebrium serve start` command, it will start up an instance and create your environment. ie: install all your requested packages and dependancies.
 
-To stop a served instance, you can simply press `Ctrl+C` in the terminal where the server is running. This will gracefully shut down the server and terminate the served instance.
+Once completed, it will output a URL that you can use to query your instance locally - mimicing a production endpoint.
 
-When a file changes in your served instance, we'll automatically sync the changes to the server and apply the changes!
-![Example changing code with serve](/images/serve/ModifyingServeCode.gif)
+```bash
+Info:   🏗️  Starting served session...
+🆔 Serve ID: dev-p-f94734f3-8-cpu-only-a296
+🔄 Syncing files
+⬆️  Uploading to Cerebrium...
+100%|████████████████████████████████████████████████████████████████████████████| 2.89k/2.89k [00:00<00:00, 68.2kB/s]
+✅ Resources uploaded successfully.
+Info:   Session dev-p-f94734f3-8-cpu-only-a296 started successfully
+
```

Comment:
Not the worst but do we want that project ID there? Would at least drop the dev part
```suggestion
🆔 Serve ID:  p-1a2b3c4d-stable-diffusion-a296
🔄 Syncing files
⬆️  Uploading to Cerebrium...
100%|████████████████████████████████████████████████████████████████████████████| 2.89k/2.89k [00:00<00:00, 68.2kB/s]
✅ Resources uploaded successfully.
Info:   Session p-1a2b3c4d-stable-diffusion-a296 started successfully

```